src/App.tsx

```diff
@@ -1,61 +1,159 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(searchParameter: number | string): User | null {
+  const foundUserById
+  = usersFromServer.find(user => user.id === searchParameter);
```

Comment:
In my opinion, this syntaxes looks better
```suggestion
  const foundUserById = usersFromServer.find(
    user => user.id === searchParameter
  );
```